onnxruntime/core/optimizer/transpose_optimizer/api_impl.h

```diff
@@ -10,6 +10,15 @@
 using namespace ONNX_NAMESPACE;
 using namespace ::onnxruntime::common;
 
+namespace onnx_layout_transformation {
+inline std::unordered_set<std::string_view> GetLayoutSensitiveOps() {
+  // Per ONNX standard resize is not a layout sensitive op. However both CPU EP and NNAPI treat it as layout sensitive,
+  // so adding it to the list of layout senstive ops.
+  return {"Resize", "Conv", "QLinearConv", "FusedConv", "AveragePool", "QLinearAveragePool", "GlobalAveragePool",
```

Comment:
nit: should api.h or transpose_optimizer.cc define the default list based on the ONNX spec, and we append Resize to that (given the Resize handling seems to be ORT specific)? the code in api.h could default to the default list for layout_sensitive_ops. 

also could the ORT specific list be a static in api_impl.cc (declare fn in .h , implement in .cc with a static function local variable for the set, construct set once from default list + Resize) so that we're not inlining construction of the set in multiple places and increasing binary size due to that?

Reply:
I also thought about this. Right now too many ops in this list are ort specific example FusedConv, QLinearAveragePool. I suggest we defer this for later when we are ready to move this to a common place.